examples/word_count/Makefile

```diff
@@ -15,20 +16,20 @@ docker-image:
 
 
 wordcount:
-	pachctl create repo urls
-	cd data && pachctl put file urls@master -f Wikipedia
-	pachctl create pipeline -f pipelines/scraper.json
-	pachctl create pipeline -f pipelines/map.json
-	pachctl create pipeline -f pipelines/reduce.json
+	pachctl create repo urls --project $(PROJECT)
```

Comment:
Realistically, it feels like we got the UI wrong for pachctl if every command is going to need a `--project` flag.  Kubernetes puts the namespace in the config file.  We probably should too.  Nobody would tolerate typing `kubectl get pods --namespace my-app` every single time.

Reply:
We store the current project in the config file, too: the user may set the default project with `pachctl config update context --project $whatever`.

Reply:
We could update the default with `pachctl config update context --project`, but I don't know if we want to update someones default project in an example makefile. It probably wouldn't be an issue though. If you prefer setting it in the context let me know and I'll do that.